Xamarin.Forms.Core/Device.cs

```diff
@@ -8,13 +8,30 @@ namespace Xamarin.Forms
 {
 	public static class Device
 	{
+		public const string iOS = "iOS";
+		public const string Android = "Android";
+		public const string WinPhone = "WinPhone";
+		public const string Windows = "Windows";
+
```

Comment:
Hello, This is Tizen . Please don't forget me.

 `public const string Tizen = "Tizen";`